src/backend/distributed/metadata/metadata_cache.c

```diff
@@ -83,7 +83,9 @@
 #include "utils/memutils.h"
 #include "utils/palloc.h"
 #include "utils/rel.h"
+#if PG_VERSION_NUM < PG_VERSION_16
```

Comment:
Probably we don't need these macros, for lower versions, it will be redundant include.

Reply:
I don't prefer redundant includes to be honest. It is easy to remove the macros when we drop support in the future.